### DISCUSSION TITLE
Images in posts

### DISCUSSION THREAD

**Discussion by jeckman** [Automattic/wordpress-activitypub#657] [General]
I'm using the plugin here: https://goatless.org/@john 

I'd like a single image to be included in each blog post 

I've tried a couple of options. Currently I've got "Post Content" set to "Custom" with this template: 

```
<p><strong>[ap_title]</strong></p>

[ap_image]
[ap_excerpt]

<p>[ap_hashtags]</p>

<p>[ap_shortlink]</p>
```

Media attachments set to 1, and "Activity-Object-Type" set to "Note"

What's the best way to ensure that most folks subscribing to my posts get the first image?

**Comment by mediaformat** [comment]
Looks like it is working, perhaps it could be optimized, I'm not sure if the features image needs to be included with a token (currently showing up as a bare url)
![image](https://github.com/Automattic/wordpress-activitypub/assets/2895351/5ec7c4ee-3d72-4a21-b98e-f6880b661875)

**Comment by jeckman** [comment]
Thanks @mediaformat  - hard to tell what these posts look like for other people. Are you subscribing also using WordPress and activity pub / friends, or elsewhere?

I've subscribed from another WP blog but not seeing images there so maybe my issue is on that end?

**Comment by jeckman** [comment]
Maybe beyond the scope of this plugin, given all the Activity Pub clients and servers out there, but it is a bit opaque how to make choices about formats and what gets included for your end readers.
